src/hocs/withTickyText.js

```diff
@@ -0,0 +1,40 @@
+import { add } from 'ramda';
+import { compose, lifecycle, withState, withHandlers } from 'recompose';
+//  /////////////////////////////////////////////////////////////////////////////////////////////////
+
+// HACK: related to async react state
+let outText = '';
+let loopNum = 0;
+
+export const withTickyText = compose(
+  withState('outputText', 'setOutputText', ''),
+  withHandlers({
+    tickInputTextWithDelay: props => (letter) => {
```

Comment:
if we`ll use arrow functions without typing no need brackets in all cases with single prop (letter)

Reply:
agree